components/common/RichText.tsx

```diff
@@ -16,21 +16,28 @@ export function RichText({ post }: { post: Post.Record }) {
   for (const segment of richtext.segments()) {
     if (segment.isMention()) {
       els.push(
-        <Link href={`/repositories/${segment.mention?.did}`}>
+        <Link
+          key={`/repositories/${segment.mention?.did}`}
+          href={`/repositories/${segment.mention?.did}`}
+        >
           {segment.text}
         </Link>,
       )
     } else if (segment.isLink()) {
       els.push(
-        <a href={String(segment.link?.uri)} title={segment.text}>
+        <a
```

Comment:
unrelated but this has been throwing errors since forever and seems like it became more acute with v19.